src/main/java/com/shade/decima/ui/data/viewer/model/dmf/DMFNodeType.java

```diff
@@ -0,0 +1,22 @@
+package com.shade.decima.ui.data.viewer.model.dmf;
+
+import com.shade.util.NotNull;
+
+public enum DMFNodeType {
+    NODE("Node"),
+    MODEL_GROUP("ModelGroup"),
+    LOD("LOD"),
+    INSTANCE("Instance"),
+    MODEL("Model");
```

Comment:
I don't see any point in naming them like that. They should remain `SNAKE_CASE`.

Reply:
It's called SCREAMING_SNAKE_CASE

Reply:
`name.toLowerCase().contains ("snake_case")`

Reply:
Oh, you meant this